frontend/packages/topology/src/components/graph-view/components/nodes/build-decorators/PipelineRunDecorator.tsx

```diff
@@ -94,7 +94,4 @@ const PipelineRunDecorator: React.FC<PipelineRunDecoratorProps & StateProps> = (
   );
 };
 
-// For testing
-export const InternalPipelineRunDecorator = PipelineRunDecorator;
-
-export default connect(impersonateStateToProps)(PipelineRunDecorator);
+export const PipelineRunDecorator = connect(impersonateStateToProps)(ConnectedPipelineRunDecorator);
```

Comment:
Don't mind the reworking of how testing component is exported, but should still be a default export for the main-use component.